ckan/config/deployment.ini_tmpl

```diff
@@ -107,7 +107,7 @@ ckan.views.default_views = image_view text_view recline_view
 ckan.site_title = CKAN
 ckan.site_logo = /base/images/ckan-logo.png
 ckan.site_description =
-ckan.favicon = /images/icons/ckan.ico
+ckan.favicon = /base/images/icons/ckan.ico
```

Comment:
Thanks for this @bunnis . The actual correct path is `/base/images/ckan.ico`:

https://github.com/ckan/ckan/blob/master/ckan/public/base/images/ckan.ico

Can you amend your PR so I can merge it?